### ISSUE TITLE
Running Test within from Visual Studio

### ISSUE THREAD

**Issue by shubyarora** [garora/TDD-Katas#12]
Hi all,
I am facing an issue while running tests from within VS2013. I do not have Resharper installed so, do not DEBU/RUN from within VS2013. Is there any way to run nUnit tests from within VS2013 so, that I should not be required to run nUnit UI separately, everytime its very awkward to load dlls.

Thanks,
Shuby

CC: @garora  @marisks  @justinkurtz  @matjos @SeanKilleen @Hassankashi

**Comment by justinkurtz** [comment]
You should use the NUnit Test Adapter extension for visual studio. https://visualstudiogallery.msdn.microsoft.com/6ab922d0-21c0-4f06-ab5f-4ecd1fe7175d

If you're running the express version of Visual Studio, consider upgrading to the new "Community" edition which will allow you to install extensions. http://www.visualstudio.com/products/visual-studio-community-vs

**Comment by garora** [comment]
Shuby,

@justinkurtz- already replied.  I did not use that plugin so can't say
anything. But, I definitely try to find out some nuget packages for the
same so I can add in repository. In this case you need not to install any
plugin.

I am busy with other stuffs so,I added this to my todo list.

**Comment by garora** [comment]
Lets keep it open untill, I find an alternative of extensions

**Comment by shubyarora** [comment]
There is one nuget package for nunit. Plz take a look


Sent from Samsung Mobile

-------- Original message --------

**Comment by garora** [comment]
Thanks I will take a look in weekend.

**Comment by garora** [comment]
Here are the solutions for your problem:

1. VS Extension - http://nunit.org/index.php?p=vsTestAdapter&r=2.6 (suggested by @justinkurtz )
2. Add nuget package - NUnit Test Adapter for VS (https://www.nuget.org/packages/NUnitTestAdapter/) - I have added this package in current release, so, onwards no need to add any extra extension or plugin to run tests from within Visual Studio.

For more info: check Release notes of 1.0.5

Closing this issue.
